internal/service/privatelinkendpointservicedatafederationonlinearchive/data_source_privatelink_endpoint_service_data_federation_online_archive_test.go

```diff
@@ -13,6 +13,7 @@ var (
 )
 
 func TestAccNetworkPrivatelinkEndpointServiceDataFederationOnlineArchiveDS_basic(t *testing.T) {
+	acc.SkipTestForCI(t)
```

Comment:
[nit] Can we add a comment describing the reason for the skip so that we don't forget?

Reply:
ha ha, i wrote the same comment :-)

Reply:
Done! Thanks for the suggestion